react/features/base/conference/actions.js

```diff
@@ -289,6 +289,25 @@ export function createConference() {
 }
 
 /**
+ * Will try to join the conference again in case it failed earlier with
+ * {@link JitsiConferenceErrors.AUTHENTICATION_REQUIRED}. It means that Jicofo
+ * did not allow to create new room from anonymous domain, but it can be tried
+ * again later in case authenticated user created it in the meantime.
+ *
+ * @returns {Function}
+ */
+export function checkIfCanJoin() {
+    return (dispatch, getState) => {
+        const { password, authRequired }
+            = getState()['features/base/conference'];
+
+        if (authRequired) {
```

Comment:
Can we find a better name for this? it suggests a boolean... yes, I know we have joining, leaving, etc. maybe we can find a better way to represent this.